CHANGELOG.md

```diff
@@ -0,0 +1,379 @@
+## [theila 0.1.0-alpha.0](https://github.com/talos-systems/theila/releases/tag/v0.1.0-alpha.0) (2021-05-20)
+
+Welcome to the v0.1.0-alpha.0 release of theila!  
+*This is a pre-release of theila*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/theila/issues.
+
+### Welcome Theila
+
+This is the first release of the UI for Talos and Sidero.
+
+First iteration is only able to run against management cluster with Sidero installed.
+In that case it fetches servers and cluster from the management clusters, is able to
+get kubeconfig and talosconfig for the deployed clusters and can also show nodes and pods in a cluster,
+as well as services in the node.
+
+
+### Contributors
+
+* Spencer Smith
+* Andrew Rynhard
+* Andrey Smirnov
```

Comment:
I did something? 😆

Reply:
well, at least you created this repo :)

Reply:
I've updated changelog. Looks like if you don't set the previous tag it also works and the output doesn't include all dependencies updates.